docs/_documentations/overview.md

```diff
@@ -1,29 +1,83 @@
+
+Different development flows are supported, so you can choose whether you want to use a desktop-based or cloud-based IDE, and whether you want to build and run your containers on your desktop or in the Cloud.
+
+Codewind can be used to build and run Docker-containerized applications on [Kubernetes](https://kubernetes.io/) container orchestration applications, including [OpenShift](https://www.openshift.com/), without the need for in-depth knowledge of container technology.
+
+### How does Codewind help me develop containerized applications?
+Codewind improves your inner loop experience, enabling you to create a microservice quickly, rapidly iterate on changes, and make improvements to performance thanks to the following features:
+
+1. **Access to preconfigured containerized project templates** Codewind provides preconfigured, Docker-containerized [project templates](mdt-vsc-workingwithtemplates.html) covering several languages, including [Node.js](https://nodejs.dev/), [Java](https://www.java.com/), [Python](https://www.python.org/) and [Swift](https://swift.org/), and several frameworks, such as [Express](https://expressjs.com/), [Spring Boot](https://spring.io/projects/spring-boot), and [Open Liberty](https://openliberty.io/). [Quickly create and deploy microservices](https://www.youtube.com/watch?v=zKMggp10gq4&t=12s) in languages and frameworks that you're familiar with. Modify the preconfigured projects to develop your customised microservice.
+
+2. **Ability to add your own standardised project templates** Create your own [project templates](mdt-vsc-workingwithtemplates.html) and your development teams can access them within their IDE. 
+
+3. **Ability to add your own applications** Access the seamless container development and deployment experience on your IDE that Codewind offers for your current projects.
+
+4. **Auto-build-and-run on code changes** As you develop, Codewind automatically pushes your code changes to your container as efficiently as possible. 
+
+5. **Ability to toggle between auto-build and manual build settings** Have control over when you want to implement Codewind's auto-build feature.
+
+6. **Project details displayed in a user-friendly panel on your IDE** Use Codewind's Project Overview interface to instantly view the endpoint, container ID, location on disk, build and run status, and ports.
+
+7. **Access application, build and container logs and run commands from within the container without leaving your IDE** Codewind integrates with your IDE's terminal window and PROBLEMS and OUTPUT panels for a smooth test and debug experience. Additionally, you can open a shell session *inside* your container with a simple click.
```

Comment:
- Should we have a comma after "logs"? It appears that "build and container logs" is one unit, so we would want the oxford comma after "logs."